packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -164,9 +164,8 @@ class BottomNavigationBar extends StatefulWidget {
   /// former is preferred, [fixedColor] only exists for the sake of
   /// backwards compatibility.
   ///
-  /// The [showSelectedLabels] argument must be non-null.
-  ///
-  /// The [showUnselectedLabels] argument defaults to `true` if [type] is
+  /// The [showSelectedLabels] argument defaults to `true`. The
```

Comment:
This isn't strictly true (it defaults to null) and it doesn't cover what's actually happening: `widget.showSelectedLabels ?? bottomTheme.showSelectedLabels ?? true`.

Reply:
Updated comment